### PR TITLE
Remove colloquialism "hold on"

### PR DESCRIPTION
**Pull request by h4ckninja**
I'm of the thought that documentation should balance between conversational grammar and formal grammar. Hold on pushes that over the conversational boundary, in my opinion. Thoughts?

### CODE REVIEW

File: form/dynamic_form_modification.rst [hunk 1]
```diff
@@ -32,8 +32,8 @@ take a look at the :doc:`Form Events </form/events>` documentation.
 Customizing your Form Based on the Underlying Data
 --------------------------------------------------
 
-Before jumping right into dynamic form generation, hold on and recall what
```
Comment (snoek09) [inline form/dynamic_form_modification.rst]: I agree on the tone which is slightly different here compared to the rest of the documentation.
If `hold on` is removed I think `jumping right into` should also be changed as it relates to each other.

Also the double semicolon (::) should be kept before a code example.

File: form/dynamic_form_modification.rst [hunk 2]
```diff
@@ -32,8 +32,8 @@ take a look at the :doc:`Form Events </form/events>` documentation.
...
-a bare form class looks like::
```
Comment (snoek09) [inline form/dynamic_form_modification.rst]: The double semicolon (::) should be kept before a code example.

**Comment by snoek09** [comment]
:+1: 

Status: Reviewed

Thanks @micheal

File: form/dynamic_form_modification.rst (hunk 1 above)
Comment (micheal) [inline form/dynamic_form_modification.rst]: I agree. I'll think of something in place of `jumping right into` and update the PR.

File: form/dynamic_form_modification.rst (hunk 2 above)
Comment (micheal) [inline form/dynamic_form_modification.rst]: Ah I didn't know that. I'll fix it and update the PR.

**Comment by micheal** [comment]
Thanks. I've updated as per above.

**Comment by weaverryan** [comment]
Right or wrong, I prioritize language that i be energetic and engaging, above formal or informal. But, that's just my opinion, and I don't think this modification is less energetic/engaging anyways.

Thanks @micheal!

**Pull request closed without merging**